lua/telescope/previewers/init.lua

```diff
@@ -1,36 +1,280 @@
+---@tag telescope.previewers
+
+---@brief [[
+--- Provides a Previewer table that has to be implemented by each previewer.
+--- To achieve this, this module also provides two wrapper that abstract most
```

Comment:
"wrapper" should be plural?

```suggestion
--- To achieve this, this module also provides two wrappers that abstract most
```